homework/index.js

```diff
@@ -23,24 +23,147 @@
       const value = options[key];
       if (key === 'text') {
         elem.textContent = value;
+      } else if (key === 'html') {
+        elem.innerHTML = value;
       } else {
         elem.setAttribute(key, value);
       }
     });
     return elem;
   }
 
-  function main(url) {
-    fetchJSON(url, (err, repositories) => {
-      const root = document.getElementById('root');
-      if (err) {
-        createAndAppend('div', root, { text: err.message, class: 'alert-error' });
+  const renderError = error => {
+    const wrapper = document.getElementById('wrapper');
+    wrapper.innerHTML = '';
+    createAndAppend('div', wrapper, {
+      text: error.message,
+      class: 'alert',
+    });
+  };
+
+  const addRepoInfoRow = (parent, label, content) => {
+    const row = createAndAppend('tr', parent);
+    createAndAppend('th', row, {
+      text: label,
+      class: 'table-header',
+    });
+    createAndAppend('td', row, {
+      html: content,
```

Comment:
Can you make this work without reverting to using `innerHTML` in `createAndAppend`? You only need it for the `<a>` tag.

Reply:
I reverted `createAndAppend` function to its original state.